storage/local/persistence.go

```diff
@@ -154,7 +155,11 @@ func newPersistence(
 		// empty. If not, we have found an old storage directory without
 		// version file, so we have to bail out.
 		if err := os.MkdirAll(basePath, 0700); err != nil {
-			return nil, err
+			if abspath, e := filepath.Abs(basePath); e != nil {
+				return nil, errors.Wrapf(err, "cannot create parsistent path %s", basePath)
```

Comment:
@brian-brazil 
You, prometheus dev team aren't willing to use "github.com/pkg/errors"?
If you are, I'd like to create PR to add dependency.